### ISSUE TITLE
Publish .ipa in Github Releases

### ISSUE THREAD

**Issue by emanuelb** [cake-tech/cake_wallet#155]
Add .ipa file (iOS app) to the github releases as asset, which currently have only .apk file for android
https://github.com/cake-tech/cake_wallet/releases

**Comment by yeru83** [comment]
Yes please, this would be great if we could check the .ipa against the build.
